framework/log/DbTarget.php

```diff
@@ -80,13 +84,16 @@ public function export()
                     $text = VarDumper::export($text);
                 }
             }
-            $command->bindValues([
-                ':level' => $level,
-                ':category' => $category,
-                ':log_time' => $timestamp,
-                ':prefix' => $this->getMessagePrefix($message),
-                ':message' => $text,
-            ])->execute();
+            if ($command->bindValues([
+                    ':level' => $level,
+                    ':category' => $category,
+                    ':log_time' => $timestamp,
+                    ':prefix' => $this->getMessagePrefix($message),
+                    ':message' => $text,
+                ])->execute() > 0) {
+                continue;
+            }
+            throw new InvalidValueException('Unable to export log through database!');
```

Comment:
`InvalidValueException` does not seem to be appropriate here: its description said:

> represents an exception caused by a function returning a value of unexpected type

Both `true` and `false` belong to the same type - `bool`. Why such exception should be here?